src/app/public/testing/dropdown/dropdown-fixture.ts

```diff
@@ -114,7 +114,7 @@ export class SkyDropdownFixture {
       throw new Error(`There is no dropdown item at index ${index}.`);
     }
 
-    itemEls[index].click();
+    itemEls[index].querySelector('button,a').click();
```

Comment:
We technically don't require that they put a button or link in the dropdown. They could have a div, span, paragraph tag, etc with a `(click)` on it and it does work. Is there a way to select the first child of the dropdown item? Or should we just support buttons and a tags?

Reply:
It might work, but from an accessibility standpoint we should only accept `button,a` elements. See: https://github.com/blackbaud/skyux-popovers/blob/master/src/app/public/modules/dropdown/dropdown-item.component.ts#L36